pillar/vault/roles/aws.sls

```diff
@@ -125,4 +125,43 @@ vault:
       name: odl-video-service-{{ env }}
       options:
         policy: '{{ ovs_policy|json }}'
+    {% load_json as mit_open_policy %}
+    {
+      "Statement": [
+        {
+          "Resource": [
+            "arn:aws:s3:::odl-discussions-{{ env }}",
+            "arn:aws:s3:::odl-discussions-{{ env }}/*"          ],
+          "Action": [
+            "s3:HeadObject",
+            "s3:Get*",
+            "s3:List*",
+            "s3:PutObject",
+            "S3:DeleteObject",
```

Comment:
Are those actions enough or does this need more extensive ones similar to OVS?

Reply:
This should be plenty. The listing for OVS is every individual S3 permission, most of which it doesn't need.